test/3x/torch/quantization/weight_only/test_gptq.py

```diff
@@ -1,295 +1,183 @@
+            assert torch.allclose(out, self.label, atol=0.25), "Accuracy gap atol > 0.25 is unexpected."
+
+    def test_mse_search(self):
+        # use_mse_search=False
+        model = copy.deepcopy(self.tiny_gptj)
         quant_config = GPTQConfig(
-            weight_dtype="nf4", weight_group_size=8, dataloader_len=len(dataloader_for_calibration), pad_max_length=512
+            use_mse_search=False,
         )
-        quant_config.set_local("lm_head", GPTQConfig(weight_dtype="fp32"))
-        logger.info(f"Test GPTQ with config {quant_config}")
-        q_model = quantize(
-            model=model_1, quant_config=quant_config, run_fn=run_fn_for_gptq, run_args=dataloader_for_calibration
-        )
-        out1 = q_model(input)
-        self.assertTrue(torch.allclose(out1[0], out0[0], atol=1e-02))
-
-    def test_gptq_advance(self):
-        # Ported from test/adaptor/pytorch_adaptor/test_weight_only_adaptor.py
-        # TestPytorchWeightOnlyAdaptor.test_GPTQ_fixed_length_quant
-
-        dataloader = GPTQLLMDataLoader()
-        model_1 = get_gpt_j()
-        input = torch.ones([1, 512], dtype=torch.long)
-        out0 = model_1(input)
-
-        device = None
-        from neural_compressor.torch.algorithms.weight_only.gptq import DataloaderPreprocessor
-
-        dataloaderPreprocessor = DataloaderPreprocessor(
-            dataloader_original=dataloader, use_max_length=False, pad_max_length=512, nsamples=128
-        )
-        dataloader_for_calibration = dataloaderPreprocessor.get_prepared_dataloader()
-
+        model = quantize(model, quant_config, run_fn=run_fn)
+        out = model(self.example_inputs)[0]
+        atol_false = (out - self.label).amax()
+        # use_mse_search=True
+        model = copy.deepcopy(self.tiny_gptj)
         quant_config = GPTQConfig(
-            weight_group_size=8,
-            dataloader_len=len(dataloader_for_calibration),
-            act_order=True,
-            enable_mse_search=True,
-            pad_max_length=512,
+            use_mse_search=True,
         )
-        quant_config.set_local("lm_head", GPTQConfig(weight_dtype="fp32"))
-        logger.info(f"Test GPTQ with config {quant_config}")
-        q_model = quantize(
-            model=model_1, quant_config=quant_config, run_fn=run_fn_for_gptq, run_args=dataloader_for_calibration
+        model = quantize(model, quant_config, run_fn=run_fn)
+        out = model(self.example_inputs)[0]
+        atol_true = (out - self.label).amax()
+        # compare atol, this case is an ideal case.
+        assert (
+            atol_false > atol_true
+        ), "use_mse_search=True doesn't help accuracy, maybe is reasonable, please double check."
+
+    # def test_layer_wise(self):
+    #     model = copy.deepcopy(self.tiny_gptj)
+    #     quant_config = GPTQConfig(
+    #         use_layer_wise=True,
+    #     )
+    #     model = quantize(model, quant_config, run_fn=run_fn)
+    # TODO(Xin): not implemented
```

Comment:
```suggestion
    # TODO: (Xin) not implemented
```
Please use `TODO: ` as a keyword for the to-do comment.

Reply:
OK